.github/workflows/aws_tfhe_multi_bit_tests.yml

```diff
@@ -77,6 +77,12 @@ jobs:
         run: |
           AVX512_SUPPORT=ON make test_integer_multi_bit_ci
 
+      - name: Run signed integer tests
```

Comment:
I will add the key gen for the integers as well in the a final commit/edit once the current tests run (to check distribution is helping and that I did not break the AVX512 in the Makefile)